sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/CosmosSourceChangeFeedConfig.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.kafka.connect.implementation.source;
+
+import java.time.Instant;
+
+public class CosmosSourceChangeFeedConfig {
+    private final CosmosChangeFeedModes changeFeedModes;
+    private final CosmosChangeFeedStartFromModes changeFeedStartFromModes;
+    private final Instant startFrom;
+    private final int maxItemCount;
+
+    public CosmosSourceChangeFeedConfig(
+        CosmosChangeFeedModes changeFeedModes,
+        CosmosChangeFeedStartFromModes changeFeedStartFromModes,
+        Instant startFrom,
+        int maxItemCount) {
```

Comment:
We should be clear in naming that ths is NOT an enforced max item count - but just a max item count hint (we will always at least return all records of he same LSN

Reply:
updated to use maxItemCountHint